src/string_bytes.cc

```diff
@@ -87,6 +91,10 @@ class ExternString: public ResourceType {
       if (length == 0)
         return scope.Escape(String::Empty(isolate));
 
+      // Workaround to avoid process crash caused by a possible V8 bug
```

Comment:
This comment is too vague; I'd phrase it as `// V8 aborts if we try to allocate a string that is too large.` (punctuation intentional.)

Shouldn't NewFromCopy() have this logic as well?